source/particle/property/interface.cc

```diff
@@ -286,6 +286,56 @@ namespace aspect
 
 
 
+      template <int dim>
+      void
+      IntegratorProperties<dim>::initialize_one_particle_property(const Point<dim> &/*position*/,
+                                                                  std::vector<double> &data) const
+      {
+        data.resize(data.size() + n_integrator_properties);
```

Comment:
Could we fill with `numbers::signaling_nan<double>()`?

Reply:
Unfortunately no. There are situations where these properties are read and copied, e.g. when during the initial adaptive refinement the number of particles violates the minimum particles per cell limit and new particles are generated. The new particle properties need to be interpolated from the old ones (still containing the signaling nan).

Reply:
I see. Can you at least initialize with an explicit zero as second argument to `resize`?

Reply:
done